Website: Add video testimonials to homepage

eashaw opened the pull request:
https://github.com/fleetdm/confidential/issues/3785

Changes:
- Added video testimonials to the homepage.

mike-j-thomas reviewed (dismissed):
Good hustle, @eashaw. Thanks!
As a follow-up, let's remove the drop shadow on the card and add it to a hover state.

eashaw commented:
> Good hustle, @eashaw. Thanks! As a follow-up, let's remove the drop shadow on the card and add it to a hover state.

I added that change to this PR and updated the description to have the browsers I tested this change on.

mike-j-thomas reviewed (approved):
Awesome. Thanks, @eashaw!
